README.md

```diff
@@ -124,7 +107,7 @@ The Microsoft Graph SDK is open for contribution. To contribute to this project,
 
 ## 7. Supported Java versions
 
-The Microsoft Graph SDK for Java library is supported at runtime for Java 8+ and [Android API revision 26](http://source.android.com/source/build-numbers.html) and greater.
+The Microsoft Graph SDK for Java library is supported at runtime for Java 17+ and [Android API revision 32](http://source.android.com/source/build-numbers.html) and greater.
```

Comment:
The minimum Android SDK version is actually still 26. https://github.com/microsoftgraph/msgraph-sdk-java/blob/7ed0c6a4e1ea3016e9942fcd571dae0313530467/android/build.gradle#L31

And although we don't have linters setup to enforce it, the required Java runtime should still be 8, not 17, unless I'm missing something? (8 got deprecated, we're making use of newer features...)

Reply:
https://en.wikipedia.org/wiki/Java_version_history 😒 2026 at the earliest for java 8...

Reply:
https://github.com/microsoftgraph/msgraph-sdk-java-core/pull/622
https://github.com/microsoftgraph/msgraph-sdk-java-core/pull/622
Bumping these down to 11 would allow us to bump the source/target compatibility of the service library down to 11 as well. Currently we are set to 17 which is not ideal for the customers.

Reply:
https://stackoverflow.com/questions/16654951/gradle-sourcecompatibility-vs-targetcompatibility